IgyToast/Private/CKToastView.swift

```diff
@@ -105,6 +107,11 @@ final class CKToastView: NibLoadingView {
     if let fV = footerView {
       self.footerView.addSubview(fV, with: .zero)
     }
+    
+    guard let backgroundColor = backgroundColor else { return }
+    contentView.backgroundColor = backgroundColor
+    titleViewHolder.backgroundColor = backgroundColor
+    bottomCoverView.backgroundColor = backgroundColor
```

Comment:
also set color for footer view if present
```suggestion
    bottomCoverView.backgroundColor = backgroundColor
    footerView?.backgroundColor = backgroundColor
```